mediapipe/pose_tracking_dll/pose_tracking.cpp

```diff
@@ -22,13 +22,13 @@
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
 #include "mediapipe/framework/calculator_framework.h"
-#include "mediapipe/framework/formats/image_frame.h"
+//#include "mediapipe/framework/formats/image_frame.h"
 #include "mediapipe/framework/formats/image_frame_opencv.h"
 #include "mediapipe/framework/formats/landmark.pb.h"
 #include "mediapipe/framework/port/file_helpers.h"
-#include "mediapipe/framework/port/opencv_highgui_inc.h"
-#include "mediapipe/framework/port/opencv_imgproc_inc.h"
-#include "mediapipe/framework/port/opencv_video_inc.h"
+//#include "mediapipe/framework/port/opencv_highgui_inc.h"
+//#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+//#include "mediapipe/framework/port/opencv_video_inc.h"
```

Comment:
The includes can be removed if not needed.

Reply:
👍